docs/outlook/onmessagesend-onappointmentsend-events.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Handle OnMessageSend and OnAppointmentSend events in your Outlook add-in with Smart Alerts
 description: Learn about the Smart Alerts implementation and how it handles the OnMessageSend and OnAppointmentSend events in your event-based Outlook add-in.
-ms.date: 12/03/2024
+ms.date: 01/07/2024
```

Comment:
What's this date? Did you mean to change the year to 2025 too?

Reply:
Thanks for catching this! I thought I had fixed the year in the PRs, but it looks like I missed one. This date is just a placeholder. When we're ready to publish, I'll update it again.